app/views/eventDetails/regTypes.html

```diff
@@ -280,8 +280,12 @@ <h4 translate>Payment</h4>
                 <label translate>Minimum Payment to Register</label>
                 <div class="input-group">
                   <span class="input-group-addon"
-                    ><i class="fa fa-usd"></i
-                  ></span>
+                    ><i class="fa"
+                      ><span
+                        >&nbsp;{{conference.currency.shortSymbol}}&nbsp;</span
```

Comment:
You could probably replace `&nbsp;` with padding.

Reply:
fixed